lib/main.js

```diff
@@ -59,7 +59,7 @@ function _vaultPath(options) {
   }
 
   // Locate .env.vault
-  return `${dotenvPath}.vault`
+  return dotenvPath.endsWith('.vault') ? dotenvPath : `${dotenvPath}.vault`
```

Comment:
The current return value may lead to errors if users add `.env` in their custom path.

Reply:
Nice addition!